platforms/common/include/px4_platform_common/px4_work_queue/WorkItem.hpp

```diff
@@ -99,11 +99,11 @@ class WorkItem : public IntrusiveSortedListNode<WorkItem *>, public IntrusiveQue
 
 	void RunPreamble()
 	{
-		_run_count++;
-
-		if (_time_first_run == 0) {
+		if (_run_count == 0) {
```

Comment:
There's a potential race condition here with the console resetting _run_count after `work queue status`, but I'm not sure it's worth worrying about as it will only potentially impact the status output. I don't want to further burden the actual WorkItem execution.